tests/client/core/test_utils.js

```diff
@@ -1,8 +1,11 @@
+import url from 'url';
+
```

Comment:
It's because this lib is the same name as a stdlib module so the linter expects it to be in a group on its own.